extide/gradle/external/binaries-list

```diff
@@ -16,4 +16,4 @@
 # under the License.
 
 A271DEA739D4EBBD1A51F282AA130EB9314AD830 gradle-wrapper-4.10.2.jar
-2564503471E1FC18A4E7D2A120BC567EA3B26AA4 gradle-4.10.2-bin.zip
+E2E8BEBFCC5BA21D6FE9A27325D10EC1DB5D83FA gradle-6.7-bin.zip
```

Comment:
Is this change really compatible?

Reply:
Well this one is just to compile the netbeans-gradle-tooling. It is needed to be able to debug the tooling jar inside NetBeans when using a modern JDK. The tooling jar still on Java 8 source (and with that target) level.
Basically this is just a compile-only dependency.